src/ngraph/runtime/gpu/gpu_backend.cpp

```diff
@@ -20,11 +20,13 @@
 #include <cudnn.h>
 
 #include "ngraph/graph_util.hpp"
+#include "ngraph/runtime/cpu/cpu_backend.hpp"
```

Comment:
Don't modify the GPU backend. If you must create a wrapper which wraps the GPU/CPU/INTERPRETER in a new backend.
This change makes it a requirement that the CPU is built when the GPU is built which is very annoying for GPU developers because the CPU is very slow to build. Also, if you are going to do something like this you will need to update the cmake files to guarantee that everything that is required is built.